Testing/SystemTests/tests/framework/PolarizationCorrections/PolarizationCorrectionsBase.py

```diff
@@ -0,0 +1,59 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from abc import ABCMeta, abstractmethod
+from systemtesting import MantidSystemTest
+
+
+from mantid.api import AnalysisDataService
+from mantid.simpleapi import *
+
+
+class PolarizationCorrectionsBase(MantidSystemTest, metaclass=ABCMeta):
+    _tolerance = 1e-5
+
+    def runTest(self):
+        self._run_test()
+
+    @abstractmethod
+    def _run_test(self):
```

Comment:
This set of pre-processing steps is very specific to the polarised SANS work. Would it be clearer to name this class `SANSPolarizationCorrectionsBase`?